pkg/github/installation_token.go

```diff
@@ -34,6 +35,7 @@ func GetInstallationToken(
 	installationToken, _, err := githubClient.Apps.CreateInstallationToken(
 		context.Background(),
 		installationID,
+		&github.InstallationTokenOptions{},
```

Comment:
Here I purposefully went minimal, though there are some interesting configuration options in the future: https://developer.github.com/v3/apps/#create-an-installation-access-token-for-an-app

Reply:
I think this may be a small vulnerability. It may not be new, but we may want to close it.

This installation token gets added to a new object that wraps the original event payload and sent downstream where a worker can eventually use it. According to the API documentation, it seems that if we don't ask for this token to be restricted to a particular repository, the token may grant permissions on _all_ repositories covered by the installation. i.e. If the app is installed at the _org_ level, this token would grant permissions on all repos in that org. I don't think we want that. It would allow a worker for project x receiving events from repo org/x to do things to the org/y repo.

Reply:
Thanks @krancour , I was thinking the same.  I'm pretty confident it isn't new (since the API didn't appear to have these optional permissions provisions before, I can't see how the token didn't cover all of the repos where a given app was installed).

However, I agree, minimally getting the repo name from the event, if provided, and scoping permissions down to just this repo, would be prudent.  I'd prefer this to be a separate follow-up, if you are willing.  Further exploration on just how fine-grained we can get the permissions scoped could then be done.  In the meantime, I'd like to get a patch release out with this dependency bumped such that users can upgrade their gateways to use the latest/non-deprecated installation token endpoint.  WDYT?

Reply:
Given that it's not new, a follow-up PR seems ok.

Reply:
Ok, follow-up filed here: https://github.com/brigadecore/brigade-github-app/issues/97